app/templates/objectives/card.html

```diff
@@ -1,19 +1,26 @@
 {% macro input(title, image_num='', bullet_points=[]) -%}
-  <div class="objective-choice resource-choice card">
+<div>
+  <div class="objective-choice resource-choice card grid-x">
     <a name="objective-{{ image_num }}-content">
-        <div class="grid-y">
-          <div class="cell small-8 large-9 card-img card-img{{ image_num }}"></div>
-          <div class="cell small-4 large-3 card-text">
-            <p class="card-text-header">{{ title }}</p>
-            <div id="objective-{{ image_num }}-content" class="objective-card__content">
-              <ul>
-                  {% for item in bullet_points %}
-                    <li>{{ item }}</li>
-                  {% endfor %}
-              </ul>
-            </div>
-          </div>
+      <div class="cell card-img card-img{{ image_num }}">
+        <div class="card-img-question-mark">
+          <div class="card-img-helper"></div>
+          <img src="../../static/img/icons/icon_question_green.svg">
```

Comment:
```suggestion
          <img src="{{ url_for('static', filename='img/icons/icon_question_green.svg') }}">
```

Reply:
If you need some files from the static folder, this is the way to go in jinja templates (templates with html and pseudo code)